target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -188,6 +188,9 @@ hg255d)
 	set_wifi_led "$boardname:green:wlan"
 	ucidef_set_led_netdev "internet" "internet" "$boardname:green:internet" "eth0.2"
 	;;
+hilink,hlk-7628n)
+	set_wifi_led "hlk-7628n:green:wlan"
```

Comment:
You should follow other examples using $boardname instead. This helps reducing script length as devices with similar LED naming scheme can be merged together.
It looks like you can already merge this one with these boards:
```bash
3g-6200nl|\
skylab,skw92a|\
wnce2001)
	set_wifi_led "$boardname:green:wlan"
	;;
```

Reply:
> You should follow other examples using $boardname instead. This helps reducing script length as devices with similar LED naming scheme can be merged together.
> It looks like you can already merge this one with these boards:
> 
> ```shell
> 3g-6200nl|\
> skylab,skw92a|\
> wnce2001)
> 	set_wifi_led "$boardname:green:wlan"
> 	;;
> ```

Thanks for the suggestion，Already modified。